Rlabkey/man/labkey.domain.save.Rd

```diff
@@ -0,0 +1,38 @@
+\name{labkey.domain.save}
+\alias{labkey.domain.save}
+\title{Updates an existing LabKey domain}
+\description{
+Modify an existing domain with the specified domain design.
+}
+\usage{
+labkey.domain.save(baseUrl=NULL, folderPath, schemaName, queryName, domainDesign)
+}
+\arguments{
+  \item{baseUrl}{a string specifying the \code{baseUrl}for the labkey server}
+  \item{folderPath}{a string specifying the \code{folderPath} }
+  \item{schemaName}{a string specifying the name of the schema of the domain }
+  \item{queryName}{a string specifying the query name }
+  \item{domainDesign}{a list data structure with the domain design to update to }
+}
+\value{
+A list containing elements describing the domain after the update. The structure is the same as a domain design created by \code{labkey.createDomainDesign}
+}
+
+\author{Karl Lum}
+\seealso{
+\code{\link{labkey.domain.get}},
+\code{\link{labkey.domain.inferFields}},
+\code{\link{labkey.domain.createDesign}},
+\code{\link{labkey.domain.create}},
+\code{\link{labkey.domain.drop}}
+}
+\examples{
+\donttest{
+
+## retrieve an existing domain
+library(Rlabkey)
+
+labkey.domain.save(baseUrl="http://labkey/", folderPath="home", schemaName="lists", queryName="test list")
```

Comment:
Seems like this example is incomplete. I assume you meant to have it call the labkey.domain.get() and then update that domain and then save it?